### DISCUSSION TITLE
城市：天际线 请求更新

### DISCUSSION THREAD

**Discussion by yangfl** [Gamuxorg/bbs#199] [投递和请求游戏]
游戏名称：城市：天际线
游戏名称(英文)：Cities: Skylines
下载地址 :（如果有的话）：magnet:?xt=urn:btih:4cad552d6af7e738ea49f38246b85545469c2e13&dn=CitiesSkylines-v1.12.0-f5.tar.xz&tr=http%3a%2f%2fbt4.t-ru.org%2fann%3fmagnet

**Comment by shouhuanxiaoji** [comment]
老哥有兴趣参与维护一下么。。开放服务器上传权限给你，appiamge打包非常简单

**Comment by cyrus26** [comment]
> 

怎么打包？

**Comment by hyh19962008** [comment]
> > 
> 
> 怎么打包？

https://doc.appimage.cn/docs/home/

**Comment by liulitchi** [comment]
已更新1.14，欢迎反馈游戏体验
